src/control/common/proto/ctl/common.pb.go

```diff
@@ -18,7 +16,7 @@ var _ = math.Inf
 // is compatible with the proto package it is being compiled against.
 // A compilation error at this line likely means your copy of the
 // proto package needs to be updated.
-const _ = proto.ProtoPackageIsVersion3 // please upgrade the proto package
+const _ = proto.ProtoPackageIsVersion2 // please upgrade the proto package
```

Comment:
It seems as if your PR is downgrading a lot of these messages. You should be using protobuf 3.11.4 per the developer guide: https://daos-stack.github.io/dev/development/#protobuf-compiler

Please install the correct versions of the RPMs from this link: https://wiki.hpdd.intel.com/download/attachments/58133974/protobuf-3.11.4.tar?api=v2

You'll also need to make sure that the correct version of protoc-gen-go is installed, per the instructions on that page.

Reply:
No problem.  I'll fix it.